### PR TITLE
Nanoflann quicktest.

### PR DESCRIPTION
**Pull request by luca-heltai**
#3230

### CODE REVIEW

**Comment by davydden** [comment]
in reference to https://github.com/dealii/dealii/issues/3230

File: tests/quick_tests/nanoflann.cc [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+//
+//---------------------------------------------------------------------
+
+// Create a list of points, and compute the minimum distance from some other points
+// to this set, using a kdtree library
+
+#include <deal.II/base/logstream.h>
+#include <deal.II/numerics/kdtree.h>
+
+
```
Comment (jppelteret) [inline tests/quick_tests/nanoflann.cc]: @luca-heltai Judging by an error I got when making the Adol-C quick test, I think that you may be missing a `using namespace dealii;` here. Do you want me to add a patch to my open PR?

Comment (luca-heltai) [reply tests/quick_tests/nanoflann.cc]: yes, please do. Notice however that all  quicktests  pass on my machine...

Comment (jppelteret) [reply tests/quick_tests/nanoflann.cc]: Ok, I've pushed the patch. Strange that excluding this line from the Adol-C test resulted in an error, e.g.
```
<>/tests/quick_tests/adolc.cc:62:3: error: use of undeclared identifier 'deallog'; did you mean 'dealii::deallog'?
  deallog << "Error (function 1): " << error_func_1 << std::endl;
```
while you didn't see the same here :-/

**Pull request merged**